Tasks/ServiceFabricDeploy/utilities.ps1

```diff
@@ -185,12 +185,12 @@ function Add-Certificate
 
         if ($ConnectedServiceEndpoint.Auth.Parameters.CertificatePassword)
```

Comment:
The password is not required.  If the certificate does not require a password, you can just pass in null which is what would happen with this code.